css/index.css

```diff
@@ -1 +1,501 @@
+}
+
+.contact-content form button {
+	border: none;
+  background-color: #e8e8e8;
+  cursor: pointer;
+}
+
+.contact-content form button:hover {
+  background-color: #2b2b2b;
+  color: white;
+}
+
+.contact-content form input::placeholder,
+.contact-content form textarea::placeholder {
+  color: #555;
+}
+/* end of Contact page */
+
+/* Tablet */
+@media (max-width: 800px) {
+  .container {
+    max-width: 758px;
+    width: 100%;
+  }
+
+  /* Header */
+  header {
+    flex-direction: column-reverse;
+    align-items: center;
+  }
+
+  header nav {
+    width: 100%;
+  }
+
+  header .logo {
+    margin-left: 8%;
+  }
+
+  /* cta */
+  .cta img {
+    display: none;
+  }
+
+  .cta .cta-text {
+    width: 100%;
+  }
+
+  .cta .cta-text button {
+    padding: 8px 5.8%;
+  }
+
+  /* Main content */
+  .main-content .top-content,
+  .main-content .bottom-content {
+    margin: 0 1.23%;
+  }
+
+  .main-content .top-content .text-content {
+    width: 48%;
+  }
+
+  .main-content .bottom-content .text-content {
+    width: 29%;
+  }
+
+  /* Contact */
+  .contact {
+    margin-left: 1.23%;
+  }
+
+  /* Services Page */
+  .services-content .card {
+    width: 100%;
+    margin: 10px;
+  }
+
+  .services-content .card button {
+    padding: 10px 6%;
+  }
+
+  .services-content .services-info .facts {
+    width: 44.5%;
+    margin: 0;
+    padding-right: 4.25%;
+  }
+}
+
+/* Mobile */
+@media (max-width: 500px) {
+  .container {
+    max-width: 465px;
+    width: 100%;
+  }
+
+  /* Header */
+  header nav {
+    width: 100%;
+    margin-top: 24px;
+    border-top: 1px solid #777;
+    flex-direction: column;
+  }
+
+  header nav a {
+    width: 100%;
+    border-bottom: 1px solid #777;
+    padding: 25px 0;
+    text-align: center;
+    font-size: 2rem;
+  }
+
+  header nav a:hover {
+    text-decoration: none;
+    background: #2b2b2b;
+    color: #fff;
+  }
+
+  /* Main content */
+  .main-content {
+    padding: 45px 0;
+  }
+
+  .main-content .top-content,
+  .main-content .bottom-content {
+    margin: 0;
+  }
+
+  .main-content .top-content {
+    height: 230px;
+    margin-top: 10px;
+    align-items: flex-end;
+  }
+
+  .main-content .top-content .text-content,
+  .main-content .bottom-content .text-content {
+    width: 100%;
+  }
+
+  .main-content .top-content .text-content:first-child {
+    align-self: flex-start;
+  }
+
+  .main-content .bottom-content {
+    flex-direction: column;
+    height: 373px;
+  }
+
+  /* Contact */
+  .contact {
+    margin-left: 0;
+  }
+
+  /* Services Page */
+  .services-content .services-info {
+    padding: 40px 2% 0 2%;
+  }
+
+  .services-content .services-info .facts {
+    width: 100%;
+    margin: 0;
+    padding: 0;
+  }
+
+  .services-content .services-info .services-info-img {
+    width: 100%;
+    height: auto;
+  }
+
+  /* Contact page */
+  .contact-content form input,
+  .contact-content form textarea,
+  .contact-content form button {
+    width: 80%;
+  }
+}
+
+/* Animation */
```

Comment:
I love these animations! They really make the user experience so much better! The fade is really nice 😍